### PR TITLE
Fixed the error when bbox dimensions gets less than 0 or greater than 1

### PR DESCRIPTION
**Pull request by POONAM2015**
image

### CODE REVIEW

**Comment by Dipet** [comment]
We can not change bboxes silently, because these changes might be incorrect in some cases. Better to add flag `clamp_bbox=False` into this function and into [`BboxParams and BboxProcessor`](https://github.com/albumentations-team/albumentations/blob/master/albumentations/core/bbox_utils.py#L65)

**Comment by POONAM2015** [comment]
I have added the parameter `clamp_bbox=False` to `BBoxParams` which will control whether bboxes will be forcefully clamped to remain between 0 and 1.
Thanks.

File: albumentations/core/bbox_utils.py [hunk 1]
```diff
@@ -118,7 +119,7 @@ def check(self, data: Sequence, rows: int, cols: int) -> None:
         check_bboxes(data)
 
     def convert_from_albumentations(self, data: Sequence, rows: int, cols: int) -> List[BoxType]:
-        return convert_bboxes_from_albumentations(data, self.params.format, rows, cols, check_validity=True)
+        return convert_bboxes_from_albumentations(data, self.params.format, rows, cols, check_validity=True, clamp_bbox=self.params.clamp_bbox)
 
     def convert_to_albumentations(self, data: Sequence[BoxType], rows: int, cols: int) -> List[BoxType]:
         return convert_bboxes_to_albumentations(data, self.params.format, rows, cols, check_validity=True)
```
Comment (Dipet) [inline albumentations/core/bbox_utils.py:125]: You added this parameter to wrong function. You changed `to` function, but added param to `from` function. Please add small test to check that this functionality works coreectly

**Comment by ternaus** [comment]
Looks like the problem could be resolved with `clip=True` passed to BboxParams

**Pull request closed without merging**